node/core/dispute-coordinator/src/real/spam_slots.rs

```diff
@@ -90,14 +98,16 @@ impl SpamSlots {
 		let validators = self.unconfirmed.entry((session, candidate)).or_default();
```

Comment:
I'm a little confused why we add unconfirmed to `self.unconfirmed.entry` but we do the check above against `self.slots.entry`. Can you explain?